### PR TITLE
feat(*): add ci for beamcoder

### PR DESCRIPTION
**Pull request by stepancar**
add CI for beamcoder, similar to framefusion.

### CODE REVIEW

**Comment by stepancar** [comment]
@animanathome , do you remember, publishing of this package is just yarn publish, o did you build something locally?

**Comment by antoineMoPa** [comment]
> @animanathome , do you remember, publishing of this package is just yarn publish, o did you build something locally?

I think I just published manually, I thought it would just be a temporary repo until https://github.com/Streampunk/beamcoder/pull/113 would be merged. Sadly, I lost hope that the original repo would be maintained at this point.

**Comment by antoineMoPa** [comment]
@stepancar do you think we should take more ownership of supporting beamcoder and updating it to newer ffmpeg version?  Or you see this repo as temporary before we move on to something like webcodecs?

File: .circleci/.github/workflows/publish-npm.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# This workflow will publish npm package on release
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
```
Comment (antoineMoPa) [inline .circleci/.github/workflows/publish-npm.yml:7]: @stepancar can we have it publish when we merge to main branch? I feel like it's a lot of bureaucracy to have to create a release after merging.

**Comment by stepancar** [comment]
> @stepancar do you think we should take more ownership of supporting beamcoder and updating it to newer ffmpeg version? Or you see this repo as temporary before we move on to something like webcodecs?

I see this repo temporary.
I just wanted to fix it and publish, but it's published under your npm account. I decided to move it here in order to reduce entropy

File: .circleci/.github/workflows/publish-npm.yml (hunk 1 above)
Comment (stepancar) [reply .circleci/.github/workflows/publish-npm.yml:7]: @antoineMoPa , let's skip it for now? I tried to make it similar to what we have for framefusion

Comment (antoineMoPa) [reply .circleci/.github/workflows/publish-npm.yml:7]: ok, makes sense to keep similar and eventually move everything at once to a new workflow.

**Review by antoineMoPa — APPROVED** [review]
LGTM!

**Pull request merged**